src/Resources/skeleton/registration/twig_template.tpl.php

```diff
@@ -1,6 +1,13 @@
 <?= $helper->getHeadPrintCode('Register'); ?>
 
 {% block body %}
+    {% for flashError in app.flashes('verify_email_error') %}
+        <div class="alert alert-danger" role="alert">{{ flashError }}</div>
+    {% endfor %}
+    {% for message in app.flashes('success') %}
+        <section class="flash-success"><p>{{ message }}</p></section>
+    {% endfor %}
```

Comment:
As we talked about, this just won't work. At this moment (and I go back and forth), we should:

A) Set the success message into `success`
B) On finish of the maker, give them a TODO to print that in their base layout or tweak that part of the controller. Heck, we could even check the `base.html.twig` to see if it looks like they're looping over a `success` flash and omit that message if we think it's there.

Reply:
I took the easy approach for now - 
1) added a todo in the template to change the redirect route and handle / remove the flash on successful verification

2) maker command shows a hint in the success message